CefSharp/ModelBinding/IPropertyInterceptor.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CefSharp.ModelBinding
+{
+    /// <summary>
+    /// Provides the capability intercepting get/set property calls made from javascript as part of the
+    /// JavascriptBinding (JSB) implementation.
+    /// </summary>
+    public interface IPropertyInterceptor
+    {
+        /// <summary>
+        /// Called before the get property is invokved. You are now responsible for evaluating
+        /// the property and returning the result.
+        /// </summary>
+        /// <param name="propertyGetter">A Func that represents the property to be called</param>
+        /// <param name="propertName">Name of the property to be called</param>
+        /// <returns>The property result</returns>
+        /// <example>
+        /// 
+        ///public object IPropertyInterceptor.InterceptGet(Func<object> propertyGetter, string propertyName)
```

Comment:
I doubt this is valid xml doc as it contains angle brackets for the `Func`. 

Either encode the brackets or put the code in a `<code/>` block with `CDATA`

```c#
/// <example>
/// <code>
/// <![CDATA[
/// public object IPropertyInterceptor.InterceptGet(Func<object> propertyGetter, string propertyName)
/// {
///    object result = propertyGetter();
///    Debug.WriteLine("InterceptGet " + propertyName);
///    return result;
/// }
/// ]]>
/// </code>
/// </example>
```